plugins/appwrite/test-fixtures/import_prefs.json

```diff
@@ -0,0 +1,4 @@
+{
+  "key": "zsaugacpwq6k54nnbdbmh1cys98u2a32qqkacma2ioxn1e2j6eyrk9urom0vzcvm6qbbm8s6l4xbm86n37foauiqba9tlcvohuoz87j7nwvpob5wr71k58i105fn39a10vj7ob84opwf1vrfat3m8konch7xxy2z2dh1ykohdbef7xgmvtn82lebe4mzmfzoylqy4jslrok11zbjtmd6xs84ukd7b1k9ofyuanvinmlhkgua32p5x0gqbexample",
```

Comment:
Is this key the same as what's stored in API_KEY?

Reply:
Yes, the format is same.